src/main/resources/lang/zh_cn/messages.yml

```
@@ -371,4 +371,12 @@ PCB_RULE_PEER_PROGRESS_CHEAT_TESTING: "Peer 作弊快速测试"
 IPDB_RETRY_WITH_BACKUP_SOURCE: "IPDB 下载任务失败或超时，正在使用备用源重试……"
-IPDB_UNGZIP_FAILED: "IPDB 数据库 {} 解压失败"
\ No newline at end of file
+IPDB_UNGZIP_FAILED: "IPDB 数据库 {} 解压失败"
+DOWNLOADER_BC_LOGIN_FAILED: "登录到 {} 失败：{} - {}: {}"
+DOWNLOADER_BC_FAILED_REQUEST_TORRENT_LIST: "请求 Torrents 列表失败 - {} - {}"
+DOWNLOADER_BC_FAILED_REQUEST_PEERS_LIST_IN_TORRENT: "请求 Torrent 的 Peers 列表失败 - {} - {}"
+DOWNLOADER_BC_API_PREFERENCES_ERR: "BitComet API 返回了非预期响应 - {} - {}"
+DOWNLOADER_BC_FAILED_SAVE_BANLIST: "无法保存 {} ({}) 的 Banlist！{} - {}\n{}"
+DOWNLOADER_BC_CONFIG_IP_FILTER: "检测到 BitComet IpFilter 配置不正确，正在自动修正……"
+DOWNLOADER_BC_CONFIG_IP_FILTER_SUCCESS: "BitComet IpFilter 配置自动修正成功"
+DOWNLOADER_BC_CONFIG_IP_FILTER_FAILED: "BitComet IpFilter 配置自动修正失败"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)